temporarily use users-api as baseurl for testbed-routes: for demo

LauriGofore opened the pull request.

LauriGofore committed:
temporarily use users-api as baseurl for testbed-routes: for demo

lsipii commented on packages/af-shared/src/lib/api/endpoints/index.ts:
```diff
@@ -20,6 +20,10 @@ export const USERS_API_BASE_URL = process.env.NEXT_PUBLIC_USERS_API_BASE_URL
 
-export const JMF_SKILL_RECOMMENDATIONS_ENDPOINT = process.env.JMF_SKILL_RECOMMENDATIONS_ENDPOINT 
+// @TODO: revert back
+export const TESTBED_API_BASE_URL = USERS_API_BASE_URL;
```
Users api:lla on productizereille urlissa nimiavaruus `/productizer` eli konsti olisi:

```
`${USERS_API_BASE_URL}/productizer`
```

LauriGofore replied:
Ihmettelin että miten tämä pelaili suoraan ilman tuota muutosta, mutta tuohan lueskellaan nyt ihan eri muuttujasta, eli `process.env.TESTBED_PRODUCT_GATEWAY_BASE_URL;`

lsipii replied:
Totta joo, tämä nyt muokattu on sille "isExported"-appikselle (joka käyttää tosiaan sitä rustilla tehtyä testbed-apia) ja se envistä tuleva on api-appikselle.

lsipii replied:
Voisi jossain kohti vaikka nimetä ne eriksi

LauriGofore replied:
Jeps, muutin nyt tuonne kontin enviin tuon, menisikö tuolla? Vaihtoehtosesti voisi sinne docker-filuun tehdä muutos, tai pulumin conffiin ja siitä vaan lukea, ei kai väliä missä se on.

lsipii replied:
Näyttäisi kiinteältä!

LauriGofore committed:
revert previous back, change TESTBED_PRODUCT_GATEWAY_BASE_URL in container env

lsipii reviewed (approved):
Mietin että tätä ei varmaan demon takia tarvitse mergata kun deplaa vaan branchin, sama urers-api:n kanssa vaikkakin siinä pitää kikkailla reveritissä (mainiin nähden) sit kantamuutoksien kanssa.

lsipii committed:
fix: interpolate pulumi output

LauriGofore committed:
Merge branch 'VFD-301-esimerkkitoteutus-kayttoehtojen-datatuotteelle' into temporary-testbed-bypass

dependabot[bot] committed:
chore(deps-dev): bump postcss from 8.4.28 to 8.4.31

Bumps [postcss](https://github.com/postcss/postcss) from 8.4.28 to 8.4.31.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.28...8.4.31)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

LauriGofore committed:
Merge pull request #64 from Virtual-Finland-Development/dependabot/npm_and_yarn/postcss-8.4.31

chore(deps-dev): bump postcss from 8.4.28 to 8.4.31

LauriGofore committed:
turbo update

dependabot[bot] committed:
chore(deps): bump zod and next

Removes [zod](https://github.com/colinhacks/zod). It's no longer used after updating ancestor dependency [next](https://github.com/vercel/next.js). These dependencies need to be updated together.


Removes `zod`

Updates `next` from 13.4.19 to 13.5.4
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v13.4.19...v13.5.4)

---
updated-dependencies:
- dependency-name: zod
  dependency-type: indirect
- dependency-name: next
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

LauriGofore committed:
Merge pull request #65 from Virtual-Finland-Development/dependabot/npm_and_yarn/zod-and-next--removed

chore(deps): bump zod and next

LauriGofore committed:
vfd-313

LauriGofore committed:
Merge remote-tracking branch 'origin/main' into VFD-313-tehdaan-paikka-tietosuojalausekkeen-linkille-af-sovellukseen

LauriGofore committed:
heading text change to english

LauriGofore committed:
info-navigation - map over INFO_ROUTES for more dry code, use obj values + re ove unsused import + tos-agreementactions - fix link

LauriGofore committed:
info-pages-layout: border-l only for lg

LauriGofore committed:
terminology: information security and register description -> data protection statement

LauriGofore committed:
merge from data security statement / info branch

lsipii committed:
hotfix: bad condition check for testbed schema version

lsipii committed:
Merge branch 'VFD-313-tehdaan-paikka-tietosuojalausekkeen-linkille-af-sovellukseen' into temporary-testbed-bypass

lsipii committed:
fix: add missing from merge

lsipii committed:
feat: add more informative error message when schema version is invalid

lsipii committed:
feat: add more informative error message when schema version is invalid

LauriGofore committed:
tos page - fix link and terminology to data protection statement

LauriGofore committed:
Merge branch 'VFD-313-tehdaan-paikka-tietosuojalausekkeen-linkille-af-sovellukseen' into temporary-testbed-bypass
